src/EditorFeatures/Core/RenameTracking/RenameTrackingTaggerProvider.cs

```diff
@@ -78,10 +78,9 @@ internal static bool ResetRenameTrackingStateWorker(Workspace workspace, Documen
                 if (textBuffer == null)
                 {
                     var ex = new InvalidOperationException(string.Format(
-                        "document with name {0} is open but textBuffer is null. Textcontainer is of type {1}. SourceText is: {2}",
+                        "document with name {0} is open but textBuffer is null. Textcontainer is of type {1}.",
```

Comment:
This doesn't make much of a difference with the below change (and assuming the message field is cred scanned), but in general we shouldn't be reporting the full source in fault telemetry.